cmake/test-targets.cmake

```diff
@@ -93,24 +93,30 @@ endif ()
 find_program(PROVE_PATH prove)
 find_program(UNBUFFER_PATH unbuffer)
 if (PROVE_PATH)
-    set(INVOKE_TEST_ARGS --prove-tool "${PROVE_PATH}" --make-tool "${CMAKE_MAKE_PROGRAM}" --unbuffer-tool "${UNBUFFER_PATH}" --build-directory "${CMAKE_CURRENT_BINARY_DIR}")
+    set(INVOKE_TEST_ARGS --prove-tool "${PROVE_PATH}" --make-tool "${CMAKE_MAKE_PROGRAM}" --unbuffer-tool "${UNBUFFER_PATH}" --build-directory "${CMAKE_CURRENT_BINARY_DIR}" t)
     add_test(
         NAME test-perl-testsuite
         COMMAND "${CMAKE_CURRENT_SOURCE_DIR}/tools/invoke-tests" ${INVOKE_TEST_ARGS}
         WORKING_DIRECTORY "${CMAKE_CURRENT_SOURCE_DIR}"
     )
+    add_test(
+        NAME test-local-author-perl
+        COMMAND "${PROVE_PATH}" xt
+        WORKING_DIRECTORY "${CMAKE_CURRENT_SOURCE_DIR}"
+    )
 else ()
     message(STATUS "Set PROVE_PATH to the path of the prove executable to enable running the Perl testsuite.")
 endif ()
 
 # add build system targets for invoking specific tests
+add_custom_target(test-local-author-perl COMMAND ${CMAKE_CTEST_COMMAND} -R "test-local-author-perl" WORKING_DIRECTORY "${CMAKE_CURRENT_BINARY_DIR}")
 add_custom_target(test-local COMMAND ${CMAKE_CTEST_COMMAND} -R "test-local-.*" WORKING_DIRECTORY "${CMAKE_CURRENT_BINARY_DIR}")
 add_custom_target(test-doc COMMAND ${CMAKE_CTEST_COMMAND} -R "test-doc-.*" WORKING_DIRECTORY "${CMAKE_CURRENT_BINARY_DIR}")
 add_custom_target(test-installed-files COMMAND ${CMAKE_CTEST_COMMAND} -R "test-installed-files" WORKING_DIRECTORY ${CMAKE_BINARY_DIR})
 if (PROVE_PATH)
     add_custom_target(test-perl-testsuite
-        COMMAND "${CMAKE_CURRENT_SOURCE_DIR}/tools/invoke-tests" ${INVOKE_TEST_ARGS}
-        WORKING_DIRECTORY "${CMAKE_CURRENT_SOURCE_DIR}"
+        COMMAND ${CMAKE_CTEST_COMMAND} -R "test-perl-testsuite"
```

Comment:
I think this was an oversight, as `invoke-tests` was already specified in the `add_test(est-perl-testsuite ...)` above, so I replaced it.

Reply:
No, this was no oversight. The idea was to avoid the additional "ctest layer" in case we only need to invoke prove anyway. CTest has the same role as prove - it is a test runner (although probably a more generic one). Making CTest invoke prove is only useful when also running non-Perl tests that probably shouldn't/can't run via prove. See https://github.com/os-autoinst/os-autoinst/commit/e3032d6b1cab0090db002fd2258e9443744d8329 for further reasoning.

Reply:
Ok, I reverted it. I hope `INVOKE_TEST_ARGS` will still be set

Reply:
> I hope `INVOKE_TEST_ARGS` will still be set

Yes, the change to `INVOKE_TEST_ARGS` will still apply automatically in both cases.